src/app/(header)/_components/LinkIconText/LinkIconText.tsx

```diff
@@ -0,0 +1,47 @@
+import { Icon } from '@/components';
+import { Color } from '@/types';
+import { IconName } from '@/types';
+import classNames from 'classnames';
+import Link from 'next/link';
+import './index.scss';
+
+type LinkIconTextProps = {
+  link: string;
+  iconName: IconName;
+  background: Color;
+  color: Color;
+  isFilled?: boolean;
+  children: React.ReactNode;
+};
+
+export default function LinkIconText({
+  link,
+  iconName,
+  background,
+  color,
+  isFilled,
```

Comment:
```suggestion
  isFilled = false,
```

`isFilled`가 선택 속성이니까 기본값으로 false로 주는 건 어떨까요?? ㅎㅎㅎ 아무것도 안 넘겨준다면 `undefined` 값을 가지게 되어 아래의 `<Icon name={iconName} color={color} isFilled={isFilled} />` 을 호출할 때 `undefined` 값이 넘어갈 것 같은데,,,, 맞는지 잘 모르겠네요 😅

Reply:
undefined가 넘어가는게 맞고 결국 falsy라 같은 의미일것 같긴합니다..!